packages/edit-post/src/index.js

```diff
@@ -90,15 +90,42 @@ export function initializeEditor(
 		( canInsert, blockType ) => {
 			if (
 				! select( editPostStore ).isEditingTemplate() &&
-				( blockType.name === 'core/template-part' ||
-					blockType.name === 'core/post-content' )
+				blockType.name === 'core/template-part'
```

Comment:
The comment above this on line 82 still mentions post content.

Reply:
rats, I was sure I removed that! Gone now.